packages/dialtone-vue2/recipes/conversation_view/editor/editor.vue

```diff
@@ -0,0 +1,678 @@
+        { showBtn: this.showUnderlineButton, selector: 'underline', iconName: 'underline', dataQA: 'dt-editor-underline-btn', tooltipMessage: 'Underline', onClick: this.onUnderlineTextToggle },
+        { showBtn: this.showStrikeButton, selector: 'strike', iconName: 'strikethrough', dataQA: 'dt-editor-strike-btn', tooltipMessage: 'Strike', onClick: this.onStrikethroughTextToggle },
+      ].filter(button => button.showBtn);
+    },
+
+    alignmentButtons () {
+      return [
+        { showBtn: this.showAlignLeftButton, selector: { textAlign: 'left' }, iconName: 'align-left', dataQA: 'dt-editor-align-left-btn', tooltipMessage: 'Align Left', onClick: () => this.onTextAlign('left') },
+        { showBtn: this.showAlignCenterButton, selector: { textAlign: 'center' }, iconName: 'align-center', dataQA: 'dt-editor-align-center-btn', tooltipMessage: 'Align Center', onClick: () => this.onTextAlign('center') },
+        { showBtn: this.showAlignRightButton, selector: { textAlign: 'right' }, iconName: 'align-right', dataQA: 'dt-editor-align-right-btn', tooltipMessage: 'Align Right', onClick: () => this.onTextAlign('right') },
+        { showBtn: this.showAlignJustifyButton, selector: { textAlign: 'justify' }, iconName: 'align-justify', dataQA: 'dt-editor-align-justify-btn', tooltipMessage: 'Align Justify', onClick: () => this.onTextAlign('justify') },
+      ].filter(button => button.showBtn);
+    },
+
+    listButtons () {
+      return [
+        { showBtn: this.showListItemsButton, selector: 'bulletList', iconName: 'list-bullet', dataQA: 'dt-editor-list-items-btn', tooltipMessage: 'Bullet List', onClick: this.onBulletListToggle },
+        { showBtn: this.showOrderedListButton, selector: 'orderedList', iconName: 'list-ordered', dataQA: 'dt-editor-ordered-list-items-btn', tooltipMessage: 'Ordered List', onClick: this.onOrderedListToggle },
+      ].filter(button => button.showBtn);
+    },
+
+    individualButtons () {
+      return [
+        { showBtn: this.showQuoteButton, selector: 'blockquote', iconName: 'quote', dataQA: 'dt-editor-blockquote-btn', tooltipMessage: 'Quote', onClick: this.onBlockquoteToggle },
+        { showBtn: this.showCodeBlockButton, selector: 'codeBlock', iconName: 'code', dataQA: 'dt-editor-code-block-btn', tooltipMessage: 'Code', onClick: this.onCodeBlockToggle },
+      ].filter(button => button.showBtn);
+    },
+
+    linkButton () {
+      return { showBtn: this.showAddLink.showAddLinkButton, selector: 'link', iconName: 'link-2', dataQA: 'dt-editor-add-link-btn', tooltipMessage: 'Link', onClick: this.openLinkInput };
+    },
+  },
+
+  watch: {
+    value (newValue) {
+      this.internalInputValue = newValue;
+    },
+  },
+
+  methods: {
+    onInputFocus (event) {
+      event?.stopPropagation();
+    },
+
+    removeLink () {
+      this.$refs.richTextEditor?.editor?.chain()?.focus()?.unsetLink()?.run();
+      this.closeLinkInput();
+    },
+
+    setLink (event) {
```

Comment:
It's been a while since I worked on this and don't remember many details anymore, but is there a way to bake this functionality into the existing link extension? Currently the extension is for finding and linkifying URLs in the content and this looks like when you manually add a link to some piece of text, right?